src/styles/core/typography/_font-attribute-utils.scss

```diff
@@ -94,7 +94,10 @@
 
   // generate font family
   $font-family: map-get($attributes, font-family);
-  $value: if($font-family, $font-family, -get-font-family-for-size($size));
+  $value: if($font-family,
+    $font-family,
+    var(--typography-html-font, -get-font-family-for-size($size))
```

Comment:
Do we plan to allow overriding the pre-defined `font-styles` in the future, using the `theme-settings.json` file?

Reply:
I think that might be a beneficial improvement to make in the future. Right now, I expect the main use case for typography is that users may want to adjust the two main font stacks used in the various pages, but we could always allow for more fine-grained typography overrides if we want to.